src/components/Station.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
 
-const Station: React.FC = () => {
-    const { id } = useParams();
+export interface StationProps {
+    id: string;
+    name: string;
+}
+
+const Station: React.FC<StationProps> = (attributes) => {
+    // const { id, name } = useParams();
```

Comment:
also here we want to remove commented code, it's good to give a PR  a sanity check of the changes to make sure any comments or console logs etc are removed